admin/test/services/AdminHealthcheckTest.scala

```diff
@@ -1,16 +0,0 @@
-package services
-
-import common.TestWsConfig
-import org.scalatest.{DoNotDiscover, Matchers, FlatSpec}
-import play.api.libs.ws.WS
-import test.ConfiguredTestSuite
-import scala.concurrent.Await
-import scala.concurrent.duration._
-
-@DoNotDiscover class AdminHealthCheckTest extends FlatSpec with Matchers with ConfiguredTestSuite with TestWsConfig {
-
-  "Healthchecks" should "pass" in goTo("/admin"){ browser =>
-
-    Await.result(WS.clientUrl(withHost(s"/_healthcheck"))(longTimeoutConfig).get(), 10.seconds).status should be (200)
-  }
-}
```

Comment:
All previous HealthCheckTest are redundant, we are already testing the underlying CachedHealthCheck https://github.com/guardian/frontend/blob/master/common/test/conf/CachedHealthCheckTest.scala